6-0-test-driven-development/src/test/java/com/bobocode/bst/BinarySearchTreeTest.java

```diff
@@ -0,0 +1,22 @@
+package com.bobocode.bst;
+
+import com.bobocode.util.ExerciseNotCompletedException;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+import static org.assertj.core.api.Assertions.fail;
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+class BinarySearchTreeTest {
+
+    /**
+     * Always failing test, remove it before doing the exercise
+     */
+    @Test
+    @Order(1)
```

Comment:
@maxstasiuk92 any improvement is appreciated. And going down the road **please make sure you follow the convention**. It's better when all exercises within a module follow the same structure. For instance, **other TDD exercises don't have this one failing test**. Therefore it's better to either add such a test for every exercise or to none.